web/packages/teleterm/src/ui/Search/useSearch.ts

```diff
@@ -88,9 +88,9 @@ export function useResourceSearch() {
         }
         case 'preview': {
           // In preview mode we know that the user didn't specify any search terms. So instead of
-          // fetching all 100 resources for each request, we fetch only a bunch of them to show
+          // fetching all 100 resources, we fetch only a bunch of them to show
           // example results in the UI.
-          limit = 5;
+          limit = 20;
```

Comment:
We used to have 5 for each supported resource type, which technically gives 20, yeah.

However, `rankResults` is always going to return 10 results max anyway. What do you think about extracting the magic number from `rankResults` to a const (`maxRankedResults`?) and reusing it here?